server/src/projects/projects.controller.ts

```diff
@@ -23,13 +23,14 @@ export const PROJECT_ATTRS = [
   'dcp_visibility',
   '_dcp_applicant_customer_value',
   'dcp_dcp_project_dcp_projectapplicant_Project',
-  'packages',
 ];
 
 @UseInterceptors(new JsonApiSerializeInterceptor('projects', {
   id: 'dcp_name',
   attributes: [
     ...PROJECT_ATTRS,
+
```

Comment:
@allthesignals  Can you explain why you broke this out?

Reply:
@godfreyyeung because `packages` isn't a real CRM field. All exported _ATTRS really must be real life CRM fields.

This is enforced additionally through the way we add _ATTRS into the `$select=` parts of the queries (for speed improvements).

Reply:
Oh, you mean it's a relationship instead of a static property?